Frontend/src/cse-ui-kit/footer/footer-Styled.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export type FooterProps = {
+    variant?: string;
+}
+export const StyledFooter = styled.footer<FooterProps>`
+    background: #5282FF;
+    padding: 0.25rem;
+    display: inline-grid;
+    text-align: right;
+    width: 80%;
+    bottom: 0;
+    position: fixed;
+    left: 10%;
```

Comment:
might need more padding for footer, since it doesnt seem padded enough on the top according to the screenshot